modules/convert/issue_comment.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package convert
+
+import (
+	"code.gitea.io/gitea/models"
+	api "code.gitea.io/gitea/modules/structs"
+)
+
+// ToComment converts a models.Comment to the api.Comment format
+func ToComment(c *models.Comment) *api.Comment {
+	return &api.Comment{
+		ID:       c.ID,
+		Poster:   ToUser(c.Poster, false, false),
```

Comment:
why set false directly ?

Reply:
to format al Posters of Comments same way?
I yould add an "Doer" argument to ToComment witch pass things to ToUser ...